public/js/sserve/controllers.js

```diff
@@ -18,7 +18,7 @@ projectMigrationApp.controller('projectMigrationController', ['Projects','Projec
     }
     // whether the current user is a member of the admin group or n0t
     var checkIsAdminUserUrl = $rootScope.urls.checkIsAdminUser;
-    Projects.checkIsAdminUser(checkIsAdminUserUrl).then(function(result) {
+    Projects.checkIsAdminUser(checkIsAdminUserUrl+window.location.search).then(function(result) {
```

Comment:
I would put a space before and after the "+" so that it is apparent it is a concatenation.